install.sh

```diff
@@ -358,6 +358,9 @@ else
 
   # Make the git default branch error messages go away.
   git config --global init.defaultBranch main
+  # Help handle situations where GitHub is down.
+  git config --global http.lowSpeedLimit 1000
+  git config --global http.lowSpeedTime 5
```

Comment:
What happens if the internet itself is down?

Reply:
That timeout should still hit!